[2023-03-21T14:56:09.471] [ERROR] error - ReferenceError: a is not defined
    at getUserById (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/controller/user.js:21:5)
    at dispatch (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:42:32)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-swagger-decorator@1.8.6/node_modules/koa-swagger-decorator/lib/wrapper.ts:46:9
    at Generator.next (<anonymous>)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-swagger-decorator@1.8.6/node_modules/koa-swagger-decorator/dist/wrapper.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-swagger-decorator@1.8.6/node_modules/koa-swagger-decorator/dist/wrapper.js:4:12)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-swagger-decorator@1.8.6/node_modules/koa-swagger-decorator/lib/wrapper.ts:31:5
    at dispatch (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:42:32)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/@koa+router@10.1.1/node_modules/@koa/router/lib/router.js:372:16
[2023-03-21T17:04:41.632] [ERROR] error - UnauthorizedError: Authentication Error
    at Object.throw (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/context.js:97:11)
    at jwt (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-jwt@4.0.4/node_modules/koa-jwt/lib/index.js:23:22)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-unless@1.0.7/node_modules/koa-unless/index.js:29:12
    at dispatch (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:42:32)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:34:12
    at Application.handleRequest (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/application.js:182:12)
    at Server.handleRequest (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/application.js:157:19)
    at Server.emit (events.js:400:28)
    at parserOnIncoming (_http_server.js:900:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:127:17)
[2023-03-21T17:06:03.638] [ERROR] error - UnauthorizedError: Authentication Error
    at Object.throw (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/context.js:97:11)
    at jwt (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-jwt@4.0.4/node_modules/koa-jwt/lib/index.js:23:22)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-unless@1.0.7/node_modules/koa-unless/index.js:29:12
    at dispatch (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:42:32)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:34:12
    at Application.handleRequest (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/application.js:182:12)
    at Server.handleRequest (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/application.js:157:19)
    at Server.emit (events.js:400:28)
    at parserOnIncoming (_http_server.js:900:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:127:17)
[2023-03-21T17:09:23.191] [ERROR] error - UnauthorizedError: Authentication Error
    at Object.throw (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/context.js:97:11)
    at jwt (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-jwt@4.0.4/node_modules/koa-jwt/lib/index.js:23:22)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-unless@1.0.7/node_modules/koa-unless/index.js:29:12
    at dispatch (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:42:32)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/app.js:15:9
    at dispatch (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:42:32)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:34:12
    at Application.handleRequest (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/application.js:182:12)
    at Server.handleRequest (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/application.js:157:19)
    at Server.emit (events.js:400:28)
[2023-03-21T17:12:43.556] [ERROR] error - TypeError: ctx.throw is not a function
    at AuthError.throwErr (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/utils/error.js:8:3)
    at exports.NO_AUTH (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/config/httpRes.js:16:73)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/app.js:18:9
[2023-03-21T17:12:53.661] [ERROR] error - UnauthorizedError: Authentication Error
    at Object.throw (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/context.js:97:11)
    at jwt (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-jwt@4.0.4/node_modules/koa-jwt/lib/index.js:23:22)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-unless@1.0.7/node_modules/koa-unless/index.js:29:12
    at dispatch (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:42:32)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/app.js:16:9
    at dispatch (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:42:32)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa-compose@4.1.0/node_modules/koa-compose/index.js:34:12
    at Application.handleRequest (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/application.js:182:12)
    at Server.handleRequest (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/application.js:157:19)
    at Server.emit (events.js:400:28) {
  code: 4001,
  flag: 'ErrorClass'
}
[2023-03-21T17:13:12.425] [ERROR] error - UnauthorizedError: 暂无权限
    at Object.throw (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/context.js:97:11)
    at AuthError.throwErr (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/utils/error.js:8:3)
    at exports.NO_AUTH (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/config/httpRes.js:16:73)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/app.js:18:9 {
  code: 4001,
  flag: 'ErrorClass'
}
[2023-03-21T17:14:34.059] [ERROR] error - UnauthorizedError: 暂无权限
    at Object.throw (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/node_modules/.pnpm/koa@2.14.1/node_modules/koa/lib/context.js:97:11)
    at AuthError.throwErr (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/utils/error.js:9:3)
    at exports.NO_AUTH (/Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/config/httpRes.js:16:73)
    at /Users/yuanfacang/Documents/kaiyuan/docDemo/koa-app/app.js:18:9 {
  code: 4001,
  flag: 'ErrorClass'
}
